backends/edk2-compat/edk2-svc-read.c

```diff
@@ -600,8 +589,8 @@ static int getSizeFromSizeFile(size_t *returnSize, const char* path)
 	close(fptr);
 	// turn string into base 10 int
 	*returnSize = strtol(c, NULL, 0); 
-	//strol likes to return zero if there is no conversion from string to int
-	//so we need to differentiate an error from a file that actually contains 0
+	// strol likes to return zero if there is no conversion from string to int
+	// so we need to differentiate an error from a file that actually contains 0
```

Comment:
I see some of these got fixed in the future.

General comment for this file: since it looks like most of these changes were minor to `edk-svc-read.c`, can this just get merged into the previous commit?

The commit message says it ports read and write, but it appears only `-read` and `-validate`(?) have changes.